linkerd/stack/Cargo.toml

```
@@ -16,15 +16,7 @@ linkerd-error = { path = "../error" }
 pin-project = "1"
 tokio = { version = "1", features = ["time"] }
 tracing = "0.1"
-
-[dependencies.tower]
-version = "0.4"
-# disable tower's tracing `log` integration for performance reasons, since we
-# will consume tower's traces as traces.
-default-features = false
-features = [
-    "util",
-]
+tower = { version = "0.4.2", features = ["util"]}
```

```suggestion
tower = { version = "0.4.2", features = ["util"] }
tracing = "0.1"
```

This suggestion may not apply cleanly, though, since the `tracing` line wasn't edited.